XenAdmin/Wizards/BootModesControl.cs

```diff
@@ -29,54 +29,61 @@
  * SUCH DAMAGE.
  */
 
+using System.ComponentModel;
 using System.Windows.Forms;
 using XenAPI;
 using BootMode = XenAdmin.Actions.VMActions.BootMode;
 
 namespace XenAdmin.Wizards
 {
-    public partial class BootModesControl : UserControl
-    {
-        public BootModesControl()
-        {
-            InitializeComponent();
-        }
+	public partial class BootModesControl : UserControl
+	{
+		public BootModesControl()
+		{
+			InitializeComponent();
+		}
 
-        public VM TemplateVM
-        {
-            get { return _templateVM; }
-            set
-            {
-                if (_templateVM == value)
-                    return;
+		[DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]

```

Comment:
When I drag the BootModesControl from Visual Studio Toolbox
The designer will automatically generate a redundant line
>this.bootModesControl1.TemplateVM = null;
After setting this attribute to hidden, the designer won't generate above line